studio/components/ui/MultiSelect/index.tsx

```diff
@@ -146,9 +145,41 @@ export default function MultiSelect({
               className={[
                 'flex w-full flex-wrap items-start gap-1.5 p-1.5 cursor-pointer',
                 `${selectedOptions.length === 0 ? 'h-9' : ''}`,
+                `${
```

Comment:
There's no need for this. Switch the `[].join(' ')` for `cn([...])` with the removed classes and it will work.